### ISSUE TITLE
No keyboard navigation for the GroupablePanelUnit

### ISSUE THREAD

**Issue by skamradt** [ignuicould/FluidVCL#2]
There should be some method to navigate between and within a groupable panel unit group.  Tab/shift tab should move between groups (if set as a tab stop), up/down left/right should navigate between items in the group. There should be a way to add a shortcut key to a specific groupable panel control. This will all help accessibility.  An example of this would be using windows 11 settings, navigate to a page with a list of items, say the system tab, and press the tab key until the "cursor" is in the list of panels.  up/down allows moving in the list, space performs an onClick. I'm thinking that if a group panel contains controls that have a TabSTop set, then tab from that panel should go immediately to the first child control.
